text/0059-token-metadata-document.md

```diff
@@ -0,0 +1,504 @@
+|                                                                    | France             | -127,54 €  |
+| The negative sign before the number but behind the currency symbol | Denmark            | kr-127,54  |
+| The negative sign after the number                                 | Netherlands        | € 127,54-  |
+| Enclosed in parentheses.                                           | US                 | ($127.54)  |
+
+Most currencies use the same decimal and thousands separator that other numbers in the locale use, but this is not always true. In some places in Switzerland, they use the period as a decimal separator for Swiss francs (Sfr. 127.54), but then use commas as the decimal separator everywhere else (127,54).
+
+We take the point of view that these differences are not "token metadata", which are determined by the ID of the token that we are using, but rather localization properties determined by the locale of the user. So, for example the same app might be capable of displaying `kr 127,54` and `127.54 kr`, depending on the locale of the user, and token metadata does not influence this.
+
+This is also consistent with [how shopify handles currency localization](https://polaris.shopify.com/foundations/formatting-localized-currency).
+
+# Reference-level explanation
+[reference-level-explanation]: #reference-level-explanation
+
+The token-metadata json fields have the following semantics:
+
+* `schema_version`: An integer describing the version number of the schema of this document. This is 1 until a future MCIP proposes schema modifications.
+* `signing_timestamp`: A string representation of the unix timestamp at the time of creating the document's signature.
+* `token_id`: A decimal string representation of the `u64` corresponding to the token id.
+* `currency_name`: A UTF-8 string which is the full, official name of the currency. This is intended to be an english-language noun or noun phrase.
+* `ticker_symbol`: A ticker symbol for the currency. This is appropriate to be displayed by an exchange, as a short form of the name of the currency. Not more than 12 printable ASCII characters, including `[A-Za-z0-9].-`.
+* `symbol`: Most fiat currencies have a printable character such as $, £, ¥. Some cryptocurrencies do also, Bitcoin has ₿, and Ethereum has Ξ. This may be optionally specified as a UTF-8 string in the "symbol" field. How exactly it is displayed may be locale specific and we do not attempt to formally specify that at this time.
+* `decimals`: An integer specifying how the `u64` integer in a TxOut in the blockchain is scaled to compute a user-displayed amount of the currency.
+* `logo_svg`: An optional logo image. This is a base64-encoded SVG document. This is expected to have been sanitized using something like [svg-hush](https://github.com/cloudflare/svg-hush), to remove scripting, hyperlinks to other documents, and references to cross-origin resources. The full extent of such sanitization will not be specified here.
```

Comment:
I think it's better to leave it as logo_svg, it will help readers of code using this to understand what form the logo takes. Its also possible we will revisit logo formatting and end up with several optional logo fields.